README.md

```diff
@@ -69,7 +70,14 @@ Here are pre-requisites for your project before adding to the Lightning EcoSyste
    ```
 1. Finally, create a draft PR to the repo!
 
-(Optional). \[wip\] join our Slack channel to be notified if your project is breaking
+### Additional suggestions and engagement rules
+
+- For qualification to use GPU machine we set min number of GitHub starts as 100+ (please note it s for capacity reasons and we this threshold may change in the future :)
+- (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
+- (**Kind request**) include Lightning badge in your readme:
+  ```md
+  [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
+  ```
```

Comment:
```suggestion
- For the qualification to use GPU machine we set min number of GitHub stars as 100+ (please note it is for capacity reasons and this threshold may change in the future :)
- (**Optional**) Join our [Slack](https://join.slack.com/t/pytorch-lightning/shared_invite/zt-12iz3cds1-uyyyBYJLiaL2bqVmMN7n~A) channel `#alerts-ecosystem-ci` to be notified if your project is breaking
- (**Kind request**) Include [Lightning badge](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white) in your readme:
  ```md
  [![Lightning](https://img.shields.io/badge/-Lightning-792ee5?logo=pytorchlightning&logoColor=white)](https://pytorchlightning.ai)
  ```
```

Reply:
not sure what is the change and seems GH marked it as outdated, could you please check it again...

Reply:
some typos, can't remember. Grammarly suggested some... looks like they are resolved now